alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/AlertPagedModel.java

```diff
@@ -24,17 +24,29 @@
 
 import java.util.List;
 
+import org.springframework.data.domain.Page;
+
+import net.minidev.json.annotate.JsonIgnore;
+
 public class AlertPagedModel<M extends AlertSerializableModel> extends AlertSerializableModel {
+    public static final String DEFAULT_PAGE_NUMBER = "0";
+    public static final String DEFAULT_PAGE_SIZE = "10";
```

Comment:
isn't 10 a little low? I feel we should be able to get at least 100 items a page rather quickly

Reply:
The client of the API can override this, but 10 is the default for audit entries and Black Duck API pages, so I was trying to key off of that.